bin/ci-test

```diff
@@ -35,7 +35,7 @@ function build() {
   export DOCKER_BUILDKIT=1
   export DOCKER_SCAN_SUGGEST=false
   export CORE=dependabot/dependabot-core
-  export CI=dependabot/dependabot-core-ci
+  export CI=dependabot/dependabot-updater
 
   set -x
   docker build -qt $CORE .
```

Comment:
The next line down from here references Dockerfile.ci which was removed in #6418. I think if we change it to Dockerfile.updater that should fix things?

Reply:
This script is not actually used in CI, but I changed it because I noticed the out of date reference. I will fix the one you mention too!

I think the problem may be related to this old PR I created a while ago (from which I lost context now but feels like it could be related, looking at the failures we get now:

> cd: /home/dependabot/dependabot/python: No such file or directory

Reply:
Ah yeah the path is wrong, there's two dependabots in /home/dependabot/dependabot/python, should only be one!